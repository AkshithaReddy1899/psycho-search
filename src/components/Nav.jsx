import React from 'react';
import { Link } from 'react-router-dom';
import { HashLink } from 'react-router-hash-link';

const Nav = () => {
	return (
		<div className="p-8">
			<Link to="/" className="mx-8">Home</Link>
			<Link to="/list" className="mx-8">Browse</Link>
			<HashLink to="/#map" className="mx-8">Map</HashLink>
		</div>
	);
};

export default Nav;
