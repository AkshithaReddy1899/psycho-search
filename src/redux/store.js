import {
  legacy_createStore as createStore,
  combineReducers,
  applyMiddleware,
  compose,
} from 'redux';
import thunk from 'redux-thunk';
import logger from 'redux-logger';
import personsReducer from './person';

const rootReducer = combineReducers({
  person: personsReducer,
});

const store = createStore(rootReducer, compose(applyMiddleware(logger, thunk)));

export default store;
