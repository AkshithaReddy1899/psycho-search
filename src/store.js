import { configureStore } from '@reduxjs/toolkit';
import personSlice from './slices/personSlice';

const reducer = {
  person: personSlice,
};

const store = configureStore({
  reducer,
  devTools: true,
  middleware: (getDefaultMiddleware) => getDefaultMiddleware({
    serializableCheck: false,
  }),
});

export default store;
