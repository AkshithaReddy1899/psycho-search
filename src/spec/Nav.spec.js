import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import matches from './test-utils';
import store from '../redux/store';
import Nav from '../components/Nav';

describe('Home', () => {
  it('Home matched snapshot', () => {
    matches(<Provider store={store}><BrowserRouter><Nav /></BrowserRouter></Provider>);
  });
});
