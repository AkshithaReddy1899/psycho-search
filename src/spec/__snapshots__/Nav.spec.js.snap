// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Home matched snapshot 1`] = `
<div
  className="p-8"
>
  <a
    className="mx-8 text-lg"
    href="/"
    onClick={[Function]}
  >
    Home
  </a>
  <a
    className="mx-8 text-lg"
    href="/list"
    onClick={[Function]}
  >
    Browse
  </a>
  <a
    className="mx-8 text-lg"
    href="/#map"
    onClick={[Function]}
  >
    Map
  </a>
  <a
    className="mx-8 text-lg"
    href="/disclamer"
    onClick={[Function]}
  >
    Disclamer
  </a>
</div>
`;
